src/components/WalletUnlock/LedgerNano.tsx

```diff
@@ -53,7 +53,7 @@ class LedgerNanoSDecryptClass extends PureComponent<Props, State> {
       return <UnsupportedNetwork walletType={translateRaw('x_Ledger')} network={network} />;
     }
 
-    if (!process.env.BUILD_ELECTRON && window.location.protocol !== 'https:') {
+    if (!process.env.IS_ELECTRON && window.location.protocol !== 'https:') {
```

Comment:
Either `if (!IS_ELECTRON ...)` or `if (!process.env.TARGET_ENV==='electron').

And reading this:
`if (!process.env.IS_ELECTRON && window.location.protocol !== 'https:')` It seems to mean: "when we're in the browser on not in production"?
 Since we use - or should use - `https` on dev, staging, and prod I don't see when this applies.

Reply:
>  ...when we're in the browser and not in production...

yup

> Since we use - or should use - https on dev, staging, and prod I don't see when this applies.

Local builds without https. This is error handling - Ledger doesn't work over non-https sites